core/domain/suggestion_services.py

```diff
@@ -308,6 +310,198 @@ def get_suggestion_by_id(
     return get_suggestion_from_model(model) if model else None
 
 
+@overload
+def get_translation_contribution_stats_models(
+    stats_ids: List[str], *, strict: Literal[True]
+) -> List[suggestion_models.TranslationContributionStatsModel]: ...
+
+
+@overload
+def get_translation_contribution_stats_models(
+    stats_ids: List[str]
+) -> List[Optional[suggestion_models.TranslationContributionStatsModel]]: ...
+
+
+@overload
+def get_translation_contribution_stats_models(
+    stats_ids: List[str], *, strict: Literal[False]
+) -> List[Optional[suggestion_models.TranslationContributionStatsModel]]: ...
+
+
+def get_translation_contribution_stats_models(
+    stats_ids: List[str], strict: bool = True
+) -> Sequence[Optional[suggestion_models.TranslationContributionStatsModel]]:
+    """Finds translation contribution stats by the IDs.
+
+    Args:
+        stats_ids: list(str). The IDs of the stats.
+        strict: bool. Whether to fail noisily if no stat with given ids exists.
+
+    Returns:
+        list(TranslationContributionStatsModel|None). The corresponding
+        translation contribution stats for the given IDs.
+
+    Raises:
+        Exception. The stats models do not exist for the given IDs.
+    """
+
+    stats_models = (
+    suggestion_models.TranslationContributionStatsModel.get_multi(
+        list(stats_ids)))
+
+    if strict:
+        for model in stats_models:
+            if model is None:
+                raise Exception(
+                    'The stats models do not exist for the given IDs.')
+
+    return stats_models
+
+
+@overload
+def get_translation_review_stats_models(
+    stats_ids: List[str], *, strict: Literal[True]
+) -> List[suggestion_models.TranslationReviewStatsModel]: ...
+
+
+@overload
+def get_translation_review_stats_models(
+    stats_ids: List[str]
+) -> List[Optional[suggestion_models.TranslationReviewStatsModel]]: ...
```

Comment:
Ditto as above, this should not be optional.

Reply:
And ditto below.

Reply:
Done